stories/module-adsense-setup.stories.js

```diff
@@ -77,351 +61,299 @@ const accountCompleteSettings = {
 };
 
 storiesOf( 'AdSense Module/Setup', module )
-	.add( 'Loading', () => {
-		filterAdSenseSetup();
-
-		const setupRegistry = ( registry ) => {
-			registry.dispatch( STORE_NAME ).receiveGetSettings( defaultSettings );
-			registry.dispatch( STORE_NAME ).receiveIsAdBlockerActive( false );
-		};
-
-		return <Setup callback={ setupRegistry } />;
+	.addDecorator( ( storyFn ) => {
+		global._googlesitekitLegacyData.setup.moduleToSetup = 'adsense';
+		const registry = createTestRegistry();
+		registry.dispatch( CORE_MODULES ).receiveGetModules( [
```

Comment:
I think it would be worth switching this and similar calls to using the new `provideModules` utility function.